.github/workflows/deployment.yaml

```diff
@@ -1,29 +1,35 @@
-name: Deploy to ECS
+name: Terraform (Plan||Apply)
```

Comment:
I feel extremely squidgy about putting plan and apply together in the same action. If we mess up a conditional somewhere, damage can result. Is there a reason why these actions are not maintained separately, for safety reasons?

Reply:
I don't mind separating it out.

Reply:
✅